index.js

```diff
@@ -212,14 +212,26 @@ async function installBundler(bundlerVersionInput, platform, rubyPrefix, engine,
 }
 
 async function bundleInstall(platform, engine, version) {
-  if (!fs.existsSync('Gemfile')) {
-    console.log('No Gemfile, skipping "bundle install" and caching')
-    return
+  if (await bundleInstallSpecific('gems.rb', 'gems.locked', platform, engine, version)) {
+    return true
+  }
+
+  if (await bundleInstallSpecific('Gemfile', 'Gemfile.lock', platform, engine, version)) {
```

Comment:
It seems we missed that `process.env['BUNDLE_GEMFILE']` should be used if set here too in https://github.com/ruby/setup-ruby/pull/74.

Would you mind handling that case too? Otherwise we can do it after, but it interacts with those conditions.
Probably the priority should be BUNDLE_GEMFILE if set, gems.rb if exists, Gemfile if exists.

Reply:
Actually it's simpler than that, and the logic can be simply (just 2 `if` like currently):
```js
const gemfile = process.env['BUNDLE_GEMFILE'] || 'Gemfile'
if (await bundleInstallSpecific(gemfile, `${gemfile}.lock`, platform, engine, version)) {
   ...
```
because if `$BUNDLE_GEMFILE` is set, we assume it correct and don't attempt to fallback to `Gemfile`.

Reply:
That makes sense, I'm happy to add support for it. Sorry, I was completely worn out yesterday, so I'll take a look at it now.